package.json

```diff
@@ -98,6 +98,7 @@
   "dependencies": {
     "@aragon/os": "^4.4.0",
     "@openzeppelin/contracts": "3.4.0",
+    "@openzeppelin/contracts-4": "npm:@openzeppelin/contracts@4.4.1",
```

Comment:
Oh, I understand the point - we can't plug in different version of OpenZepplin without this workaround. I just want to suggest to make it more clear to the reader and rename it like `@openzeppelin/contracts-v4.4`? What do you think?

Reply:
Ok, I take up the challenge.  Going with `contracts-v4.4` name since the post-review commits bunch.